tests/unit/cartography/intel/okta/test_admin_roles.py

```diff
@@ -0,0 +1,32 @@
+from cartography.intel.okta.oktaintel import transform_group_roles_data
+from cartography.intel.okta.oktaintel import transform_user_roles_data
+from tests.data.okta.adminroles import LIST_ASSIGNED_GROUP_ROLE_RESPONSE
+from tests.data.okta.adminroles import LIST_ASSIGNED_USER_ROLE_RESPONSE
+
+
+def test_transform_user_roles():
+    org_id = "lyft"
+    props_list = transform_user_roles_data(LIST_ASSIGNED_USER_ROLE_RESPONSE, org_id)
+
+    assert len(props_list) == 2
+    assert props_list[0]["label"] == "Application Administrator"
+    assert props_list[0]["type"] == "APP_ADMIN"
+    assert props_list[0]["id"] == "{}-{}".format(org_id, props_list[0]["type"])
```

Comment:
Hardcode what we expect the result to be. Duplicating the application logic increases the likelihood of including a bug in the test.